projects/plugins/search/readme.txt

```diff
@@ -35,16 +35,16 @@ People on eCommerce sites are 2x more likely to purchase something when they sea
 Jetpack Search is a completely customizable WordPress site search plugin, so your visitors get a search experience that blends in seamlessly with your site design.
 
 **WHY CHOOSE JETPACK SEARCH?**
-Highly relevant search results organized by modern ranking algorithms
-Boosted and prioritized results based on your site’s search stats
-Instant search and filtering without reloading the page
-Site search filters and facets (by categories, tags, dates, custom taxonomies, and post types)
-Improved theme compatibility for both mobile and desktop
-Real-time indexing, so your search index is updated within minutes of changes to your site
-Integrates seamlessly with WooCommerce
-Support for all languages, and advanced language analysis for 29 languages
-Highlighted search terms on comments and post content
-Fast and accurate spelling correction
+- Highly relevant search results organized by modern ranking algorithms
+- Boosted and prioritized results based on your site’s search stats
+- Instant search and filtering without reloading the page
+- Site search filters and facets (by categories, tags, dates, custom taxonomies, and post types)
+- Improved theme compatibility for both mobile and desktop
+- Real-time indexing, so your search index is updated within minutes of changes to your site
+- Integrates seamlessly with WooCommerce
+- Support for all languages, and advanced language analysis for 29 languages
+- Highlighted search terms on comments and post content
+- Fast and accurate spelling correction
```

Comment:
You would get an even better display by using `*` like so:

```suggestion

* Highly relevant search results organized by modern ranking algorithms
* Boosted and prioritized results based on your site’s search stats
* Instant search and filtering without reloading the page
* Site search filters and facets (by categories, tags, dates, custom taxonomies, and post types)
* Improved theme compatibility for both mobile and desktop
* Real-time indexing, so your search index is updated within minutes of changes to your site
* Integrates seamlessly with WooCommerce
* Support for all languages, and advanced language analysis for 29 languages
* Highlighted search terms on comments and post content
* Fast and accurate spelling correction
```

Here is how things will look like:

- With `-`:
<img width="828" alt="Screen Shot 2022-05-31 at 10 04 52" src="https://user-images.githubusercontent.com/426388/171124168-0b735afa-2f56-48ab-a64a-7f477b6be367.png">

- With `*`:
<img width="804" alt="Screen Shot 2022-05-31 at 10 05 01" src="https://user-images.githubusercontent.com/426388/171124207-0b2dfc3a-15af-48ec-8a31-4ad3a27b842d.png">

Reply:
That's what I was after. Thanks @jeherve!